md_api/src/main/java/com/wipro/ats/bdre/md/api/ProcessLog.java

```diff
@@ -72,7 +70,6 @@ public ProcessLogInfo getLastValue(String pid, String msgId, String logCategory)
             processLogInfo.setMessageId(msgId);
             //Calling proc GetLastValue
             processLogInfo = processLogDAO.getLastValue(processLogInfo);
-//            processLogInfo = s.selectOne("call_procedures.GetLastValue", processLogInfo);
             return processLogInfo;
         } catch (Exception e) {
             LOGGER.error("Error occurred", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Error occurred" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)